app/assets/javascripts/admin/addon/components/ace-editor.hbs

```diff
@@ -1 +1,5 @@
-<div class="ace">{{this.content}}</div>
\ No newline at end of file
+{{#if this.isLoading}}
+  {{loading-spinner size="small"}}
+{{/if}}
+
+<div class="ace hidden">{{this.content}}</div>
```

Comment:
Can we not render rather than setting the `hidden` class?

```suggestion
{{#if this.isLoading}}
  {{loading-spinner size="small"}}
{{else}}
  <div class="ace">{{this.content}}</div>  
{{/if}}
```

Reply:
It turns out I can. Initially, I was hiding it because the theme file took time to load and I wanted to avoid the flash. Now that I am loading the theme file upfront, I don't need to hide the div anymore.